kele.el

```diff
@@ -2032,29 +2038,46 @@ NAMESPACE and CONTEXT are used to identify the resource type to query for."
           ;; TODO: Error if the port is already in use
           (port (number-to-string (read-number "Port: "))))
      (list context ns gvk name port)))
-  (let ((proc-name (format "kele: port-forward (%s/%s, %s, %s, %s)" (oref gvk kind) name context namespace port)))
-    (make-process
-     :name proc-name
-     :command (list kele-kubectl-executable
-                    "--context"
-                    context
-                    "--namespace"
-                    namespace
-                    "port-forward"
-                    (format "%s/%s" (oref gvk kind) name)
-                    port)
-     :buffer (generate-new-buffer (format " *%s*" proc-name))
-     :sentinel
-     (lambda (proc _status)
-       (let ((exit-code (process-exit-status proc)))
-         (cl-case exit-code
-           (0 (message "Successfully terminated port-forward for %s" name))
-           (9 (message "Port-forward for %s (port %s) terminated" name port))
-           (t (message "Port-forward process for %s failed with exit code %s" name exit-code)))
-         (kele--kill-process-quietly proc)))
-     :noquery t))
+  (let* ((proc-name (format "kele: port-forward (%s/%s, %s, %s, %s)" (oref gvk kind) name context namespace port))
+         (proc (make-process
+                :name proc-name
+                :command (list kele-kubectl-executable
+                               "--context"
+                               context
+                               "--namespace"
+                               namespace
+                               "port-forward"
+                               (format "%s/%s" (oref gvk kind) name)
+                               port)
+                :buffer (generate-new-buffer (format " *%s*" proc-name))
+                :sentinel
+                (lambda (proc _status)
+                  (let ((exit-code (process-exit-status proc)))
+                    (cl-case exit-code
+                      (0 (message "Successfully terminated port-forward for %s" name))
+                      (9 (message "Port-forward for %s (port %s) terminated" name port))
+                      (t (message "Port-forward process for %s failed with exit code %s" name exit-code)))
+                    (kele--kill-process-quietly proc)))
+                :noquery t)))
+    (add-to-list 'kele--active-port-forwards (list port context namespace gvk name proc))
+    proc)
   (message "[kele] Started port-forward for %s/%s (port %s)" (oref gvk kind) name port))
 
+(cl-defun kele-kill-port-forward ()
+  "Kill a port-forward process.
+
+The port-forward must have been initiated with
+`kele-port-forward'."
+  (interactive)
+  (if (= 0 (length (mapcar 'car kele--active-port-forwards)))
+      (message "[kele] No port-forwards active!")
+    (let* ((port (completing-read "Port-forward to terminate: "
+                                  (mapcar 'car kele--active-port-forwards)))
```

Comment:
Let's decorate these completions w/ the context/namespace/gvk/name of the associated resource.